addons/bestja_frontend_ucw/templates/pages/about_us.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thank you for the idea, I will discuss this with Anna and we will consider what to do with them. For now I would like to leave them where they are.